### PR TITLE
[SIG-2387] Department checkbox list

### PR DESCRIPTION
**Pull request by janjaap**
Addition of a checkbox list for departments in the user detail form.

Note that the departments cannot be set yet, because the user detail page needs to be refactored first.

### CODE REVIEW

File: src/signals/incident-management/components/CheckboxList/index.js [hunk 1]
```diff
@@ -139,7 +139,10 @@ const CheckboxList = ({
    * @returns {(Object|undefined)}
    */
   const getOption = useCallback(
-    id => options.find(option => option.id === id || option.key === id),
+    // disabling linter; a strict type check is not needed because an id with value 6 will not equal '6' and that is
+    // what we need when comparing form field values since everything from a form is a string
+    // eslint-disable-next-line eqeqeq
```
Comment (aditudorache) [inline src/signals/incident-management/components/CheckboxList/index.js]: what is eqeqew?

File: src/signals/settings/users/containers/Detail/components/UserForm/__tests__/UserForm.test.js [hunk 2]
```diff
@@ -27,14 +42,16 @@ describe('signals/settings/users/containers/Detail/components/UserForm', () => {
 
   it('should make fields disabled', () => {
     const { container, rerender, queryByText } = render(withAppContext(<UserForm />));
-    const numFields = container.querySelectorAll('input').length;
+    // const numFields = container.querySelectorAll('input').length;
```
Comment (aditudorache) [inline src/signals/settings/users/containers/Detail/components/UserForm/__tests__/UserForm.test.js]: forgot to remove?

Comment (janjaap) [reply src/signals/settings/users/containers/Detail/components/UserForm/__tests__/UserForm.test.js]: No, this still needs to be there for an upcoming PR. See comment a couple of lines down.

File: src/signals/incident-management/components/CheckboxList/index.js (hunk 1 above)
Comment (janjaap) [reply src/signals/incident-management/components/CheckboxList/index.js]: `eqeqeq` Is strict equality.

**Pull request merged**